core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/ServerConfig.java

```diff
@@ -87,4 +98,140 @@
 
   /** Attributes of the server */
   Map<String, String> getAttributes();
+
+  /**
+   * Determines whether the attributes configure the server to be internal.
+   *
+   * @param attributes the attributes with additional server configuration
+   * @see #INTERNAL_SERVER_ATTRIBUTE
+   */
+  static boolean isInternal(Map<String, String> attributes) {
+    return AttributesEvaluator.booleanAttr(attributes, INTERNAL_SERVER_ATTRIBUTE, false);
+  }
+
+  /**
+   * Sets the "internal" flag in the provided attributes to the provided value.
+   *
+   * @param attributes the attributes with the additional server configuration
+   */
+  static void setInternal(Map<String, String> attributes, boolean value) {
```

Comment:
This is done because the attributes are present not only on a `ServerConfig` but also on other objects like `ServerDto`. IMHO having a single place that contains the logic for manipulating the attributes is a good thing.

The fact that it is the static methods on a single interface is IMHO irrelevant. It could have been a static utility class, but interface is IMHO exactly that as well.

Reply:
Static methods on interfaces have IMHO been added to Java to exactly reduce the proliferation of utility classes.